site/pages/gcweb-theme/static-header-footer/bootstrap-3.html

```diff
@@ -65,14 +74,6 @@ <h2 class="wb-inv">Menu</h2>
 			</ul>
 		</div>
 	</nav>
-	<nav id="wb-bc" property="breadcrumb">
-		<h2>You are here:</h2>
-		<div class="container">
-			<ol class="breadcrumb">
-				<li><a href="https://www.canada.ca/en.html">Home</a></li>
-			</ol>
-		</div>
-	</nav>
```

Comment:
Not sure why you are getting rid of the breadcrumb. Can you please explain? Thanks.